keyboards/melgeek/mj61/rev1/rev1.c

```diff
@@ -0,0 +1,124 @@
+    {0, CS24_SW4, CS23_SW4, CS22_SW4}, /* RGB40 */
+    {0, CS24_SW5, CS23_SW5, CS22_SW5}, /* RGB41 */
+    {0, CS24_SW6, CS23_SW6, CS22_SW6}, /* RGB42 */
+    {0, CS24_SW7, CS23_SW7, CS22_SW7}, /* RGB43 */
+    {0, CS24_SW8, CS23_SW8, CS22_SW8}, /* RGB44 */
+    {0, CS24_SW9, CS23_SW9, CS22_SW9}, /* RGB45 */
+    {0, CS27_SW1, CS26_SW1, CS25_SW1}, /* RGB46 */
+	{0, CS27_SW2, CS26_SW2, CS25_SW2}, /* RGB47 */
+    {0, CS27_SW3, CS26_SW3, CS25_SW3}, /* RGB48 */
+	{0, CS27_SW4, CS26_SW4, CS25_SW4}, /* RGB49 */
+	{0, CS27_SW5, CS26_SW5, CS25_SW5}, /* RGB50 */
+    {0, CS27_SW6, CS26_SW6, CS25_SW6}, /* RGB51 */
+    {0, CS27_SW7, CS26_SW7, CS25_SW7}, /* RGB52 */
+    {0, CS27_SW8, CS26_SW8, CS25_SW8}, /* RGB53 */
+    {0, CS27_SW9, CS26_SW9, CS25_SW9}, /* RGB54 */
+    {0, CS30_SW1, CS29_SW1, CS28_SW1}, /* RGB55 */
+    {0, CS30_SW2, CS29_SW2, CS28_SW2}, /* RGB56 */
+	{0, CS30_SW3, CS29_SW3, CS28_SW3}, /* RGB57 */
+    {0, CS30_SW4, CS29_SW4, CS28_SW4}, /* RGB58 */
+    {0, CS30_SW5, CS29_SW5, CS28_SW5}, /* RGB59 */
+    {0, CS30_SW6, CS29_SW6, CS28_SW6}, /* RGB60 */
+    {0, CS30_SW7, CS29_SW7, CS28_SW7}, /* RGB61 */
+    {0, CS30_SW8, CS29_SW8, CS28_SW8}, /* RGB62 */
+    {0, CS30_SW9, CS29_SW9, CS28_SW9}, /* RGB63 */
+};
+
+led_config_t g_led_config = {
+	{
+		/* C0     C1         C2        C3        C4        C5        C6        C7        C8          C9        C10         C11        C12        C13  */
+		{  0,      1,         2,        3,        4,        5,        6,        7,        8,          9,        10,         11,       12,        13     }, /* R0 */
+		{ 27,     26,        25,       24,       23,       22,       21,       20,       19,         18,        17,         16,       15,        14     }, /* R1 */
+		{ 28,     29,        30,       31,       32,       33,       34,       35,       36,         37,        38,         39,       40,        NO_LED,}, /* R2 */
+		{ 52,     NO_LED,    51,       50,       49,       48,       47,       46,       45,         44,        43,         42,       41,        NO_LED }, /* R3 */
+		{ 53,     54,        NO_LED,   55,       61,       56,       62,   NO_LED,   NO_LED,         57,        58,         59,       60,        NO_LED }, /* R4 */
+    }, {
+        {0, 0},   {16, 0},   {32, 0},  {48, 0},  {64, 0},  {80, 0},  {96, 0},  {112, 0},  {128, 0},  {144, 0},  {160, 0},  {176, 0},  {192, 0},  {208, 0},
+		{0, 16},  {16, 16},  {32, 16}, {48, 16}, {64, 16}, {80, 16}, {96, 16}, {112, 16}, {128, 16}, {144, 16}, {160, 16}, {176, 16}, {192, 16}, {208, 16},
+		{0, 32},  {16, 32},  {32, 32}, {48, 32}, {64, 32}, {80, 32}, {96, 32}, {112, 32}, {128, 32}, {144, 32}, {160, 32}, {176, 32}, {192, 32},
+		{0, 48},             {16, 48}, {32, 48}, {48, 48}, {64, 48}, {80, 48}, {96, 48},  {112, 48}, {128, 48}, {144, 48}, {160, 48}, {176, 48},
+		{0, 64},  {16, 64},            {32, 64}, {48, 64}, {64, 64}, {80, 64},                       {96, 64},  {112, 64}, {128, 64}, {144, 64},
+	}, {
```

Comment:
Wait, so you have no modifiers on your keyboard, only alphas?

Reply:
yes. all of leds for key backlight

Reply:
So no escape/tab/capslock/shift/ctrl/ etc?